script/deploy_dags/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/astronomer/ap-airflow:2.2.4-onbuild
+
+# TODO: install local code base
```

Comment:
You can use the same commands as https://github.com/astronomer/astronomer-providers/blob/e3c53bb3fa7e962b56a3d2a837ea7b480824c9ca/dev/Dockerfile#L4-L11

i.e in your bash script copy the entire `astronomer-providers` folder and run your script from root, 

```bash
cp -r ../astronomer-providers astronomer-providers
docker build -f `scripts/deploy_dags/Dockerfile` 
```

and then in Dockerfile, you can do something like:


```Dockerfile
FROM quay.io/astronomer/ap-airflow:2.2.4-onbuild
COPY astronomer-providers /tmp/astronomer-providers
pip install /tmp/astronomer-providers
```